tsp-cli-client

```diff
@@ -152,19 +155,38 @@ if __name__=="__main__":
             else:
                 sys.exit(1)
 
+        if (options.delete_trace):
+            response = tsp_client.delete_trace(options.delete_trace, False)
+            if response.status_code == 200: 
+                print("Successfully deleted trace")
+                sys.exit(0)
+            else:
+                sys.exit(1)
+
+        trace_uuids = []
         if (options.experiment):
-            if options.uuids is not None:
-                response = tsp_client.open_experiment(options.experiment, options.uuids)
-                if response.status_code == 200: 
-                    res = response.model
-                    print_experiment(res)
-                    sys.exit(0)
-                else:
-                    sys.exit(1)
+            if options.paths is not None:
+                for path in options.paths:
+                    response = tsp_client.open_trace(path, path)
```

Comment:
we should use the last segment as name otherwise the server resource path would be too long.